src/services.js

```diff
@@ -69,6 +69,15 @@ export function activityForDoc(nodeOrDoc) {
 }
 
 /**
+ * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrDoc
+ * @return {!Promise<../extensions/amp-form/0.1/amp-form.AmpFormService>}
```

Comment:
This should be `!Promise<!../...>`, right? I.e. the promise yields a non-null value...